flink/v2.0/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergSink.java

```diff
@@ -421,8 +421,9 @@ void testOperatorsUidNameWitUidSuffix() throws Exception {
 
   @TestTemplate
   void testErrorOnNullForRequiredField() throws Exception {
-    Assume.assumeFalse(
-        "ORC file format supports null values even for required fields.", format == FileFormat.ORC);
+    assumeThat(format)
+        .as("ORC file format supports null values even for required fields.")
+        .isEqualTo(FileFormat.ORC);
```

Comment:
@nastra Sorry one thing, I think this should be `isNotEqualTo` because the current one uses `assumeFalse`

Reply:
oops you're right, thanks for spotting this